src/uproot/streamers.py

```diff
@@ -815,42 +777,34 @@ def class_code(
 
         if streamerinfo.name == "TBranch" and self.name == "fBasketSeek":
             read_members.append(
-                "        if context.get('speedbump', True):\n"
-                "            if cursor.bytes(chunk, 1, context)[0] == 2:\n"
-                "                tmp = numpy.dtype('>i8')"
+                """        if context.get('speedbump', True):
+                            if cursor.bytes(chunk, 1, context)[0] == 2:
+                                tmp = numpy.dtype('>i8')"""
```

Comment:
Be careful with this and Python's indentation rules. The string that this generates is actually

```
        if context.get('speedbump', True):
                            if cursor.bytes(chunk, 1, context)[0] == 2:
                                tmp = numpy.dtype('>i8')
```

which is technically okay for indented Python but the `if cursor.bytes...` is indented 20 spaces more than the line above it, rather than 4 spaces more. For this code, that works because the indentation rule is a stack: a nested block only needs to be indented _more_ than its surroundings; it doesn't need to be only one level more.

However, if this is to be concatenated with some more code that assumes the indented block is only 4 spaces more, or if the second line wasn't an indentation of the first, then there would be Python syntax errors or wrong behavior. (This is a reason why Python isn't a good language for metaprogramming! Forth's minimal syntax allows the code that generates it to take priority in aesthetics over the code that is generated.)

How about this? It adds a carriage return at the beginning and the end so that we can write it inline, like ordinary Python with the right indentation, and the `.strip("\n")` removes the outermost `"\n"` (but none of the internal ones, and not other whitespace like `" "`).

```suggestion
                """
        if context.get('speedbump', True):
            if cursor.bytes(chunk, 1, context)[0] == 2:
                tmp = numpy.dtype('>i8')
""".strip("\n")
```